cmd/testgrid-config-generator/main.go

```diff
@@ -296,7 +296,8 @@ func main() {
 				strings.HasPrefix(name, "periodic-ci-openshift-multiarch"),
 				strings.HasPrefix(name, "periodic-ci-openshift-release-master-ci-"),
 				strings.HasPrefix(name, "periodic-ci-openshift-release-master-okd-"),
-				strings.HasPrefix(name, "periodic-ci-openshift-release-master-nightly-"):
+				strings.HasPrefix(name, "periodic-ci-openshift-release-master-nightly-"),
+				strings.HasPrefix(name, "periodic-ci-shiftstack-shiftstack-ci-main-periodic-"):
```

Comment:
I *think* you also need to add this to the switch below (L324), I think this one handles some kind of presence in the allowlist (sorry for misleading you, I don't know the generator too well).